models/classes/class.ItemsService.php

```diff
@@ -68,6 +68,8 @@ class taoItems_models_classes_ItemsService extends OntologyClassService
 
     const INSTANCE_FORMAL_PARAM_ITEM_URI = 'http://www.tao.lu/Ontologies/TAODelivery.rdf#FormalParamItemUri';
 
+    const DIV_CLASS_EMPTY = '<div class="empty"';
```

Comment:
`'<div class="empty">'` Did you do intentional to forget the `>` or don't needed?

Reply:
don't need it , and it might generate issues if we add a second class to the existing  <div class="empty" template from FE